models/all-articles-model.js

```diff
@@ -0,0 +1,23 @@
+// REQUIRE
+const db = require("../db/connection.js");
+
+//FUNCTION 
+function fetchAllArticles() {
+  return db
+    .query(
+      `SELECT articles.author, articles.title, articles.article_id, 
+      articles.topic, articles.created_at, articles.votes, 
+      articles.article_img_url, COUNT(comments.article_id) AS comment_count
```

Comment:
Cheeky SQL magic here, if you include with the count:
`COUNT(comments.article_id)::INT` 
It will give you the result in the form of a number rather than a string 🪄